modules/s3/s3codecs/pdf.py

```diff
@@ -90,6 +91,17 @@
 PDF_WIDTH = 0
 PDF_HEIGHT = 1
 
+if current.response.s3.language == "tr":
```

Comment:
This is OK as a quick hack for Turkish, but as soon as we add support for a 2nd language, this really should be a lookup.
I am sure that there will be many languages which need Unifont & a few may require other languages.
I was suggesting adding a dict of lang code -> font mappings.